menu.html

```diff
@@ -0,0 +1,138 @@
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Inter:wght@300&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="styles.css">
+  <title>Tsige's Personal Portifolio</title>
+</head>
+
+<body>
+  <section class="headline">
+   <div class="works">
+      <h2>My Recent Works</h2>
+      <img src="images/icons/Indicator.svg" alt="Indicator"><br>
+    <div class="work-list">
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>Ruby On Rails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+      <div class="grid-container1">
+        <div class="grid-item1"></div>
+        <div class="grid-item2">
+          <h4>Multi-Post Stories Gain+Glory</h4>
+          <div id="2">
+            <ul>
+              <li>RubyOnRails</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+              <li>HTML</li>
+            </ul>
+          </div> 
+          <div class="button">See project</div>
+      </div> 
+        </div>
+       </div>
+      </div>
+      <div class="About">
+        <h1>About me</h1>
+        <div> <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hesitate to contact me.</p></div>
+          <div class="btn">
+            <p>Get my resume</p>
+          </div>
+          </div>
+          <div class="lang">
+                <div class="lang-img"><img src="images/icons/language.svg" alt=""></div>
+                <div class="lang1">Languages</div>
+                <div><ul>
+                    <li>JavaScript</li>
```

Comment:
- [ ] As I suggested before, to make your code more organized and clean, the Technology categories, `Language, Framework and Skills` should be in a `ul` tag with each of them being a list item `li`. :+1: I'll suggest you update this part of the code accordingly.
- [ ] Looking at the screenshot below, You can see your project is missing the ICONS in these technology categories, I'll suggest you get them from the given design and include them in your project. :+1: :wave: 

- YOUR DESIGN:
![Screenshot from 2023-05-06 16-45-43](https://user-images.githubusercontent.com/107632780/236634080-1d1ea321-b9a7-4331-98fe-761874d95ec3.png)

- FIGMA DESIGN:
![Selection_036](https://user-images.githubusercontent.com/107632780/236634093-12f9faef-456a-4630-81f4-c80c2c6026dd.png)